week02/excersise2/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
### Error Message
![image](https://github.com/user-attachments/assets/0ed7a39f-2f69-46cf-9c97-c4127c047078)

### Solution
Add a meta description in the `<head>` tag, like this: `<meta name="description" content="[the description here]">`.